docs/user/build-customization.rst

```diff
@@ -105,24 +105,14 @@ To avoid this, it's possible to unshallow the clone done by Read the Docs:
          - git fetch --unshallow
 
 
-Skip build based on a condition
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Cancel build based on a condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There may be situations where you want to skip a build that was automatically triggered when someone on your team pushed to the repository.
-Skipping builds will allow you to speed up review times and also help us reduce server costs and ultimately our environmental footprint.
-Consider the following scenarios:
+When a command exits with code ``439``,
```

Comment:
This error code isn't a valid exit code (https://tldp.org/LDP/abs/html/exitcodes.html), we shouldn't document a code above 255 otherwise users will get confused.